charts/kyverno/templates/crds.yaml

```diff
@@ -22,1200 +21,932 @@ spec:
     listKind: ClusterPolicyList
     plural: clusterpolicies
     shortNames:
-    - cpol
+      - cpol
```

Comment:
I don't understand why there are changes in this file from the PR title. It seems to mostly be whitespace. Why was this included?

Reply:
yes, whitespace consistency for the file.

Reply:
reverted this change....!